kapitan/cli.py

```diff
@@ -276,7 +278,7 @@ def _search_imports(cwd, imp):
         compile_targets(args.inventory_path, search_paths, args.output_path,
                         args.parallelism, args.targets, ref_controller,
                         prune=(args.prune), indent=args.indent, reveal=args.reveal,
-                        cache=args.cache, cache_paths=args.cache_paths,
+                        cache=args.cache, cache_paths=args.cache_paths, force_fetch=args.fetch,
```

Comment:
where is `force_fetch` implemented?

Reply:
@ramaro Thanks for the review! It's not yet implemented, because I was unsure whether `--fetch` flag indicates force-fetch or simply fetch-then-compile.

Reply:
@ramaro I'm gonna assume `--fetch` as force fetch. I don't think there is much use in compile without fetch because that will fail always.

Reply:
@yoshi-1224 I think it's best to stick to fetch-then-compile for now. In the future we might even think about a `$ kapitan deps --fetch` option for force-fetch only. Thanks!

Reply:
@ramaro I think we posted comments at the same moment!
Ok so is it correct that without `--fetch` flag, `compile` will just run without fetching the dependencies?

Reply:
That's it :+1: